e2e/setup.go

```diff
@@ -26,6 +27,8 @@ import (
 	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
 
 	"github.com/stretchr/testify/require"
+	miniotc "github.com/testcontainers/testcontainers-go/modules/minio"
+	redistc "github.com/testcontainers/testcontainers-go/modules/redis"
```

Comment:
nice didn't even know that this existed

Reply:
yeah they have a bunch. I typically reach out for these premades when they exist.